lib/ansible/plugins/inventory/script.py

```diff
@@ -9,15 +9,20 @@
     inventory: script
     version_added: "2.4"
     short_description: Executes an inventory script that returns JSON
+    extends_documentation_fragment:
+      - inventory_cache
     options:
       cache:
         description: Toggle the usage of the configured Cache plugin.
         default: False
         type: boolean
         ini:
+           - section: inventory
+             key: cache
            - section: inventory_plugin_script
```

Comment:
Since these cache settings never worked, should I just remove this and the ANSIBLE_INVENTORY_PLUGIN_SCRIPT_CACHE env var?